quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.unix.approved.txt

```diff
@@ -150,14 +150,14 @@ HTTP(S):
 --https-protocols <protocols>
                      The list of protocols to explicitly enable. Default: TLSv1.3,TLSv1.2.
 --https-trust-store-file <file>
-                     DEPRECATED: The trust store which holds the certificate information of the
-                       certificates to trust.
+                     DEPRECATED. Use: truststore-paths. -- The trust store which holds the
```

Comment:
I'm not sure if it's worth it to convey, but the usage of truststore-paths would be optional as you could just use the defatult location.

Reply:
I'll add a more descriptive note there (`deprecatedWithNote`).

Reply:
Change the wording a bit and removed the hinted option.